.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor/
 phpunit.xml
 composer.lock
+.idea/
```

Comment:
Don't add editor/environment specific ignores to repo. They should be in your system's global gitignore.